src/modules/MeasureTool/MeasureToolCore/PerGlyphOpacityTextRender.cpp

```diff
@@ -74,7 +74,7 @@ HRESULT __stdcall PerGlyphOpacityTextRender::DrawGlyphRun(void* /*clientDrawingC
     auto opacityEffect = clientDrawingEffect.try_query<IDrawingEffect>();
 
     if (opacityEffect)
-        _baseBrush->SetOpacity(static_cast<OpacityEffect*>(opacityEffect.get())->alpha);
```

Comment:
it's impossible to fail here, since the Render itself is made to operate on such types only (`OpacityEffect`). I guess we could also change it to perform `dynamic_cast` and then `assert` it's not `nullptr`, since generally dynamic_cast can fail.